tests/openvino/test_quantization.py

```diff
@@ -82,7 +82,7 @@
 class OVQuantizerTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES_TORCH_MODEL = (
         (OVModelForSequenceClassification, "bert", 32 if is_transformers_version("<", "4.41.0") else 22, 35),
-        (OVModelForCausalLM, "gpt2", 41 if is_transformers_version("<", "4.42.0") else 21, 3),
+        (OVModelForCausalLM, "gpt2", 41 if is_transformers_version("<", "4.42.0") else 21, 23),
```

Comment:
The number of int8 constants has increased due to improvements in https://github.com/openvinotoolkit/nncf/pull/2713

cc @AlexanderDokuchaev